README.md

```diff
@@ -3,6 +3,20 @@
 This is a repository for the AI Engineering team to share code with customers and the community.  This ranges from simple example API snippets to demonstrations and end to end tutorials and associated code.
 
 
+Additional code examples can be accessed from the following locations:
+
+- [Tutorials for data scientists repo](https://github.com/datarobot-community/tutorials-for-data-scientists/)
+- [Examples for data scientists repo](https://github.com/datarobot-community/examples-for-data-scientists/)
+- [AI Engineering repo](https://github.com/datarobot-community/ai_engineering)
+- [docs.datarobot.com](https://docs.datarobot.com/en/docs/api/code-examples/index.html)
+- [R Vignettes included in the R client](https://cran.r-project.org/web/packages/datarobot/index.html)
+- [Examples included in the Python client](https://datarobot-public-api-client.readthedocs-hosted.com/en/v2.27.1/examples/index.html)
+
+## Important Links
+
+- To learn to use DataRobot, visit [DataRobot University](https://university.datarobot.com/).
+- For articles on using DataRobot, feature deep dives, and example workflows, visit [DataRobot Community](https://community.datarobot.com/).
+
 ## Usage
 
 Multiple integrations and projects will be hosted within this repo, see individual readmes in subfolders for details relevant to those individual entries.
```

Comment:
Almost all links/content in the Usage section link to nonexistent Community articles. I've asked @fhuthmacher to remove all "Related Community Articles" content" but since you're in here, can you do that? Or better would be to link instead 
![image (47)](https://user-images.githubusercontent.com/47538938/153919708-5f3ba707-d05b-47df-ad0c-5054c9ac43a9.png)
to new doc content created from those Community articles, etc. At the very least, we want to remove all those links to Community content that no longer exists.